pyiron_contrib/atomistics/atomistics/job/trainingcontainer.py

```diff
@@ -54,6 +54,10 @@ def __init__(self, project, job_name):
         self._container.add_array("forces", shape=(3,), dtype=np.float64, per="element")
         # save stress in voigt notation
         self._container.add_array("stress", shape=(6,), dtype=np.float64, per="chunk")
+        # Store total (per-structure) and atomic charge information.
+        self._container.add_array('totalcharge', dtype=np.float64, per='chunk')
+        self._container.add_array("charges", dtype=np.float64, per="element")
```

Comment:
Best to add these only once needed, otherwise this will increase storage needed.